src/commcare_cloud/ansible/roles/backups/tasks/main.yml

```diff
@@ -60,6 +60,21 @@
     - cron
     - backups
 
+- name: Create Hourly Cron job
+  become: yes
+  cron:
+    name: "Backup blobdb hourly"
+    job: "/usr/local/sbin/create_blobdb_backup.sh hourly {{ blobdb_backup_hours * 60 }}"
+    minute: 15
+    hour: "*/{{ nadir_hour_frequency|default(3) }}"
+    weekday: "1,2,3,4,5,6"
+    user: "{{ blobdb_user }}"
+    cron_file: backup_blobdb
+  when: "backup_blobdb and blobdb_backup_hourly and 'shared_dir_host' in group_names"
```

Comment:
Nit: I think instead of having one task to add and one task to remove, you can combine them by removing the when clause and having

`state: "{{ 'present' if backup_blobdb and blobdb_backup_hourly and 'shared_dir_host' in group_names else 'absent' }}"`

Reply:
@dannyroberts sure, we can do it. I will make that change.

Reply:
@dannyroberts as discussed, I have combined the task to add/remove the cron jobs in the same YAML. 
However, for Blobdb and Postgres we need to run the cleanup script to remove all cronjobs. As we discussed, the cleanup script is mainly to update the backup script and cronjobs point to the latest script so I had to include the cleanup tasks in the cleanup.yml file. Please review and let me know your suggestions.

Reply:
Gotcha. Sounds good!